.travis.yml

```diff
@@ -1,8 +1,6 @@
 dist: trusty
 language: java
 jdk:
-  - oraclejdk8
-  - openjdk8
-  - openjdk7
+  - openjdk11
```

Comment:
We could also do jdk 9 and 10

Reply:
Both of those versions are already EOL, and didn't see much adoption AFAIK. I'm not sure the intersect between people who wish to use both asciidoclet, and Java 9 or 10, and can't upgrade to Java 11, is big enough to make it worth while. How about we assume, for the time being, that this is not needed, and if that turns out to be wrong, then we can create a branch based on the Java 11 code?

Reply:
Yeah, that can work.